specs/CameraCaptureUI/CameraCaptureUI.md

```diff
@@ -0,0 +1,387 @@
+
+
+
+
+## CameraCaptureUIPhotoCaptureSettings Class
+Provides settings for capturing photos with CameraCaptureUI. 
+The settings include aspect ratio, image size, format, resolution, 
+and whether or not cropping is allowed by the user interface (UI).
+
+## CameraCaptureUIPhotoCaptureSettings Properties
+
+| Name | Description | Value |
+|-|-|-|
+| AllowCropping | Determines whether photo cropping will be enabled in the user interface for capture a photo.| Boolean |
+| CroppedAspectRatio | The aspect ratio of the captured photo.| Windows.Foundation.Size |
+| CroppedSizeInPixels |The exact size in pixels of the captured photo.| Windows.Foundation.Size |
+| Format |Determines the format that captured photos will be stored in.| CameraCaptureUIPhotoFormat |
+| MaxResolution | Determines the maximum resolution the user will be able to select. | CameraCaptureUIMaxPhotoResolution |
+
+
+
+
+## CameraCaptureUIVideoCaptureSettings Class
+Provides settings for capturing videos. 
+The settings include format, maximum resolution, maximum duration, and whether or not to allow trimming.
+
+## CameraCaptureUIVideoCaptureSettings Properties
+| Name | Description | Value |
+|-|-|-|
+| AllowTrimming | Determines whether the user can trim the captured video.| Boolean |
+| Format | Determines the format that captured videos will be stored in.| CameraCaptureUIVideoFormat |
+| MaxDurationInSeconds | The maximum duration of the video that the user can capture.| Single |
+| MaxResolution | Determines the maximum resolution the user will be able to select.| CameraCaptureUIMaxVideoResolution |
+
+
+## CameraCaptureUIMode enum
+
+Determines whether the user interface for capturing from the attached camera allows capture of photos, 
+videos, or both photos and videos.
+
+| Name | Value | Description |
+|-|-|-|
+| PhotoOrVideo | 0 | The user interface allows capture of photos and videos. |
+| Photo | 1 | The user interface allows capture of photos only.|
+| Video | 2 | The user interface allows capture of videos only. |
+
+
+## CameraCaptureUIPhotoFormat enum
+Determines the format for photos captured with CameraCaptureUI.
+
+| Name | Value | Description |
+|-|-|-|
+| Jpeg | 0 |JPEG format. |
+| Png | 1 | PNG format. |
+| JpegXR | 2 | JPEG-XR format. |
+
+
+## CameraCaptureUIVideoFormat enum
+Determines the format for saving captured videos.
+
+| Name | Value | Description |
+|-|-|-|
+| Mp4 | 0 |MP4/H.264/AAC format.|
+| Wmv | 1 | WMV format. |
+
+## CameraCaptureUIMaxPhotoResolution enum
+Determines the highest resolution the user can select for capturing photos.
+
+| Name | Value | Description |
+|-|-|-|
+| HighestAvailable | 0 | The user can select any resolution. |
+| VerySmallQvga | 1 |The user can select resolutions up to 320 X 240, or a similar 16:9 resolution. |
+| SmallVga | 2 | The user can select resolutions up to 320 X 240, or a similar 16:9 resolution.|
+| MediumXga | 3 | The user can select resolutions up to 1024 X 768, or a similar 16:9 resolution. |
```

Comment:
We probably can't change the names if we want UWP parity, but these names are kindof hard to read and understand :(.

Reply:
That would be a fun and lengthy conversation but agreed we shouldn't create the incompatibility.

Reply:
I agree, @dmachaj. The terms VGA and XGA are unclear and don't provide much insight into their values.

Keeping it the same has the advantage of minimizing the effort required for developers who are using the CameraCapture API, transition their app from UWP to WinAppSDK.